tests/widgets_chart.rs

```diff
@@ -13,6 +14,22 @@ fn create_labels<'a>(labels: &'a [&'a str]) -> Vec<Span<'a>> {
     labels.iter().map(|l| Span::from(*l)).collect()
 }
 
+fn axis_test_case<S>(width: u16, height: u16, x_axis: Axis, y_axis: Axis, lines: Vec<S>)
```

Comment:
I wonder if it really makes sense to have a dedicated function for something that is only used in few places for now. Don't you think it would be clearer and more flexible to have each test have it inline ?

Reply:
I felt that this "technical" part of the test case did not add any value in understanding what the tests did. Also, as this code was duplicated across multiple tests it felt right to extract it to a function.

But yes I suppose that this change could hinder future modifications of the tests. If you feel that this function is not worth the trouble, I can inline it back in the tests.